Collectors/UdpCollector.py

```diff
@@ -74,9 +74,14 @@ def _create_rmq_channel(self):
         """
         Create a fresh connection to RabbitMQ
         """
-        parameters = pika.URLParameters(self.config.get('AMQP', 'url'))
-        connection = pika.BlockingConnection(parameters)
-        self.channel = connection.channel()
+        try:
+            parameters = pika.URLParameters(self.config.get('AMQP', 'url'))
+            connection = pika.BlockingConnection(parameters)
+            self.channel = connection.channel()
+
+        except Exception  as e:
+            self.logger.exception('Error while connecting rabbitmq message;')
```

Comment:
If we fail to connect to the message bus, is there anything for us to do?  Is this function retried?  Should the entire collector just panic and fail?

Reply:
On the publish method, if there is an error the collector tries to connect again.